firstapp/app/src/androidTest/java/com/example/firstapp/UserCujTest.java

```diff
@@ -0,0 +1,24 @@
+package com.example.firstapp;
+
+import org.junit.Test;
+
+public class UserCujTest {
+    /*************************************************
+     USER INPUT:
+     PACKAGE - package to be used
+     pre_CUJ - preparatory actions (unrecorded)
+     post_CUJ - recorded actions
+     RECORDING_BUF:  how long to sleep in between prep and measured actions (can be 0 if no intent to record)
+     *************************************************/
```

Comment:
Odd to have this comment here.  Should put the comment on the variables themselves.  Or even better, comment on the non-test code.